tsconfig/gutenboarding.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We can't move this file to `client/landding/gutenboarding/tsconfig.json`, can we?

Reply:
We could, however this makes it handy to reference.

If this were isolated enough to live in an _app_, I'd expect it to live in the app root, but the entrypoints like gutenboarding aren't quite the same. This seemed reasonable to me, but I'm open to move it.

Reply:
I think I'd see value in co-locating it. It might not be as separate as an app (though in some respect it might be even more so than an app, e.g. in terms of depending on Calypso stuff), but if I was a developer trying to find out about the config applying to a given dir, I'd go looking in that dir :thinking:

Reply:
Moved it (d2be25912a641d1ee729f34d35d6c0e3a8c3ea68). This allows us to run it via

```
npm run typecheck -- --project client/landing/gutenboarding/
```

or simply via

```
cd client/landing/gutenboarding/
npm run typecheck
```

which I think is quite intuitive.

---

Kinda tempted to move the base config into `calypso-build` since it's the only thing left in `tsconfig/` now :thinking:

Reply:
> Moved it ([d2be259](https://github.com/Automattic/wp-calypso/commit/d2be25912a641d1ee729f34d35d6c0e3a8c3ea68)). This allows us to run it via
> 
> ```
> npm run typecheck -- --project client/landing/gutenboarding/
> ```
> 
> which I think is quite intuitive.
> 
> Kinda tempted to move the base config into `calypso-build` since it's the only thing left in `tsconfig/` now

Maybe as a follow-up, would need moving around of some excludes.